R/utils.R

```diff
@@ -220,7 +220,7 @@ opencage_format <- function(lst){
     }
   }
   else {
-    results <- NULL
+    results <- data.frame(query = as.character(lst$request$query))
```

Comment:
As mentioned, this is only necessary with the `opencage_forward` due to the implicit `stringsAsFactors = TRUE`